src/main/scala/akka/contrib/stream/pattern/reconnect/ReconnectingStreamHttp.scala

```diff
@@ -33,7 +31,7 @@ object ReconnectingStreamHttp {
       interval: FiniteDuration,
       retriesRemaining: Long,
       settings: Option[ClientConnectionSettings],
-      onConnection: (ConnectionStatus => Unit)) extends ReconnectionData[HttpConnected] {
+      onConnection: (HttpConnected => Unit)) extends ReconnectionData[HttpConnected] {
```

Comment:
sorry - you're right. In this pace we can expect connected, only the Future will get the failure (sorry - seems I'm sick with a fever again...)